docs/layers/ecs/tutorials/1password-scim-bridge.mdx

```diff
@@ -29,14 +29,14 @@ The implementation of this is fairly simple. We will generate credentials for th
       1. Click Integrations in the sidebar
       1. Select "Set up user provisioning"
       1. Choose "Custom"
-      1. You should now see the SCIM bridge credentials. We will need the "Bearer Token" for the next steps.
-      1. Save this token in a secure location (such as 1Password) for future reference
-      1. Store the credentials in AWS SSM Parameter Store. This will allow the ECS task definition to access the credentials securely.
+      1. You should now see the SCIM bridge credentials. We will need the "scimsession" and "Bearer Token" for the next steps.
+      1. Save these credentials in a secure location (such as 1Password) for future reference
+      1. Store the "scimsession" in AWS SSM Parameter Store. This will allow the ECS task definition to access the credentials securely. Then the server will ask for the bearer token using the session, which we will enter at that time.
```

Comment:
To be clear, the bearer token will come from a client side 1Password? Just clarifying since it seems like in the past this was stored in SSM, but now the `scimsession` replaced that... So there needs to be another source of truth for the `bearer-token`

Reply:
Both the scimsession and bearer token are presented as credentials at the same time. Then in this step we save both in 1pass, but only scimsession in AWS SSM. Then after we deploy everything, the service will prompt you to enter the bearer token to verify, which you can pull from 1pass whenever you saved it

Reply:
rephrased a little to help hopefully

Reply:
looks good!